std/random.d

```diff
@@ -3362,8 +3362,8 @@ if (isRandomAccessRange!Range)
     // Optionally @nogc std.random.randomCover
     // https://issues.dlang.org/show_bug.cgi?id=14001
     auto rng = Xorshift(123_456_789);
-    int[5] sa = [1, 2, 3, 4, 5];
-    auto r = randomCover(sa[], rng);
+    static immutable int[] sa = [1, 2, 3, 4, 5];
```

Comment:
why is the change from a static array to a `static int[]` necessary?

Reply:
A slice to a static array on the stack is `scope`, and `randomCover` doesn't support `scope` ranges yet. The test is `@nogc`, so an array literal can't be used.

Reply:
OK